### ISSUE TITLE
The disc could not be read

### ISSUE THREAD

**Issue by itotola** [CTurt/FreeDVDBoot#321]
I have tried seven diffrent dvd-r: s and tested a Avatar Movie (dvd) and it worked fine 
i have also tried the other ones like : Try diconnecting and pluggin back in the controller memory card resetting cleaning the laser diffrent burners turned on diagnosis and downloaded the newest version and nothing seems to work :( 
Console SCPH-90004
DVD Player 3.11E

**Comment by milecile** [comment]
i have the same issue with ps2 slim, scph75004 with DVD Player 3.11E

freedvdboot disc could not be read

im using verbatim dvd-r and tried also with different burning tools
i tried with burning speed 1x and also 4x

also searching for a fix

**Comment by itotola** [comment]
Yeah I have the same and it's annoying say if you find anything helpful!

Thanks.
